src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -278,4 +279,9 @@ public void stop()
         mFlipperSolenoid.set(Constants.kSolenoidRetract);
 
     }
+    
+    private boolean ballInPosition()
+    {
+        return mRampSensor.getDistance() == Constants.kBallInPositionThreshold;
```

Comment:
This should probably be:

```java
return mRampSensor.getDistance() <= Constants.kBallInPositionThreshold;
```

Also be sure to rename `kBallInPositionThreshold` to something like `kMaxChuteBallDistanceThreshold`.